pyactiveresource/util.py

```diff
@@ -196,7 +196,9 @@ def camelize(word):
         The modified string.
     """
     return ''.join(w[0].upper() + w[1:]
-                   for w in re.sub('[^A-Z^a-z^0-9^:]+', ' ', word).split(' '))
+        for w
+        in re.sub('[^A-Z^a-z^0-9^:]+', ' ', word.strip()
+        ).split(' '))
```

Comment:
I am not sure I agree with the multi line use here, is it really any easier to read?